src/aus/getConsentState.ts

```diff
@@ -6,6 +6,6 @@ export const getConsentState: () => Promise<AUSConsentState> = async () => {
 	const uspData = await getUSPData();
 
 	return {
-		personalisedAdvertising: uspData.uspString.charAt(2) === 'Y',
+		personalisedAdvertising: uspData.uspString.charAt(2) !== 'Y',
```

Comment:
wondering if we could have keep the same name as CCPA which is negative `doNotSell` . Not blocking of course

Reply:
The third character is either `Y`, `N` or `-`.
We want personalised advertising if it's either `N`, or `-`.

[As per the docs](https://github.com/InteractiveAdvertisingBureau/CCPA-reference-code):

> The usprivacy string as the format: `vnol` where
`v` = version (int)
`n` = Notice Given (char)
`o` = OptedOut (char)
`l` = Lspact (char)

Reply:
I think it's better to have our code match our copy (ON / OFF &larr;&rarr; `true` / `false`)

Reply:
I've clarified the logic in two steps with b86ee19.